src/Umbraco.Tests.Integration/Umbraco.Tests.Integration.csproj

```diff
@@ -38,6 +48,9 @@
     </ItemGroup>
 
     <ItemGroup>
+      <None Update="Logging\UmbracoTraceLog.UNITTEST.20181112.json">
```

Comment:
This looks like an error . Dont think any of the updates to need file is intented